Task8If/Task8If/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+
+namespace Task8If
+{
+    internal class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Вариант 21");
+            Console.WriteLine("Введите значение X");
+
+            double x  = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("F(x) = " + calculation(x),3);
+
+
+            Console.ReadKey();
+        }
+
+        static double calculation(double x)
+        {
+            if (Math.Cos(x) != 0)  
+                return Math.Tan(x);
+
+            if ((Math.Cos(x) == 0) && (Math.Sin(x) > 0))
+                return (x / Math.PI) - 0.5;
+
+            return (x / Math.PI) * -1 - 0.5;
+        }
+
+    }
```

Comment:
1.  Учитывайте приоритеты операций. Тогда можно не писать лишние скобки и использовать лишние операции.
2. Не используйте лишних пустых строк.
```suggestion
            if (Math.Cos(x) != 0)  
                return Math.Tan(x);

            if (Math.Cos(x) == 0 && Math.Sin(x) > 0)
                return x / Math.PI - 0.5;

            return -x / Math.PI - 0.5;
        }
    }
```